src/components/AppBar.tsx

```diff
@@ -61,14 +74,47 @@ export const AppBar = () => {
       style={{ zIndex: 1 }}
     >
       <div className="ml50 flex">
-        <Link to={routesEnum.landingPage}>
+        <Link to={routesEnum.landingPage} className="mx10">
           <EthLogo src={EthDiamond} alt="eth-diamond" />
-          <Heading level={4} margin="none" style={{ padding: '8px 0' }}>
+          <Heading level={4} margin="none" style={{ padding: '12px 0' }}>
             Eth2 Launch Pad{' '}
             {IS_MAINNET ? `` : ` for ${ETH2_NETWORK_NAME} testnet`}
           </Heading>
         </Link>
+        <Link to={routesEnum.acknowledgementPage} className="mx10">
+          <Heading level={4} margin="none" style={{ padding: '12px 0' }}>
+            Deposit
+          </Heading>
+        </Link>
+        <ValidatorDropdown
+          className="mx10 px0"
+          label={
+            <Heading level={4} margin="none">
+              Validator Clients
+            </Heading>
+          }
+          dropAlign={{ top: 'bottom', right: 'right' }}
+          dropContent={
+            <Box pad="small">
+              <DropdownLink to={routesEnum.prysm}>Prysm</DropdownLink>
+              <DropdownLink to={routesEnum.lighthouse}>Lighthouse</DropdownLink>
+              <DropdownLink to={routesEnum.teku}>Teku</DropdownLink>
+              <DropdownLink to={routesEnum.nimbus}>Nimbus</DropdownLink>
```

Comment:
nitpicking: let's go with alphabetical order for fairness?

```suggestion
              <DropdownLink to={routesEnum.lighthouse}>Lighthouse</DropdownLink>
              <DropdownLink to={routesEnum.nimbus}>Nimbus</DropdownLink>
              <DropdownLink to={routesEnum.prysm}>Prysm</DropdownLink>
              <DropdownLink to={routesEnum.teku}>Teku</DropdownLink>
```